tensorly/regression/tests/test_cp_plsr.py

```diff
@@ -207,3 +213,49 @@ def test_optimized_covariance(n_latent):
         )
 
     assert_allclose(max_cov, pls_cov)
+
+
+@pytest.mark.parametrize("n_response", [5, 7, 9])
```

Comment:
I don't see you using `n_response`. Can this parameterization be removed?

Reply:
I have added it in the function so it will make sense now.